src/Neo/VM/Helper.cs

```diff
@@ -54,18 +54,64 @@ public static ScriptBuilder CreateArray<T>(this ScriptBuilder builder, IReadOnly
         /// <param name="builder">The <see cref="ScriptBuilder"/> to be used.</param>
         /// <param name="map">The key/value pairs of the map.</param>
         /// <returns>The same instance as <paramref name="builder"/>.</returns>
-        public static ScriptBuilder CreateMap<TKey, TValue>(this ScriptBuilder builder, IEnumerable<KeyValuePair<TKey, TValue>> map = null)
+        public static ScriptBuilder CreateMap<TKey, TValue>(this ScriptBuilder builder, IEnumerable<KeyValuePair<TKey, TValue>> map)
+            where TKey : notnull
+            where TValue : notnull
```

Comment:
Value can not be null?

Reply:
You can't do `map[null] = null'. You could before but wouldn't work. Needs to be `StackItem.Null`